### PR TITLE
Added Queue [Python]

### LINKED ISSUE #510: Queue [Python]
#### This issue is about [Queue](http://www.cdn.geeksforgeeks.org/stack-data-structure-introduction-program/).

- [x] Issue Title is in form `Algo/DS Name [Language]`.
- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This Algo/DS is worth adding to this repository.
- [x] Assign this issue to me (I want to work on this).
	- [x] I ensure that I am not already assigned to 2 or more issues.
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

### PR DESCRIPTION
**Pull request by amineHorseman**
Fixes #510

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.

### CODE REVIEW

**Comment by singhpratyush** [comment]
Hi! Thanks for showing interest in contributing to this repository.

Please make sure that you have ticked the points mentioned in PR description correctly.

Thanks again!

File: README.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ Community (college) maintained list of Algorithms and Data Structures implementa
 | [Binary Search Tree](https://en.wikipedia.org/wiki/Binary_search_tree) | | | [:white_check_mark:](binary_search_tree/BinarySearchTree.java) | | [:white_check_mark:](binary_search_tree/binary_search_tree.go) | | |
 | [Linked List](https://en.wikipedia.org/wiki/Linked_list) | [:white_check_mark:](linked_list/linkedList.c.c) | | [:white_check_mark:](linked_list/LinkedList.java) | [:white_check_mark:](linked_list/linked_list.py) | [:white_check_mark:](linked_list/linked_list.go) | [:white_check_mark:](linked_list/linkedList.js) | [:white_check_mark:](linked_list/LinkedList.cs) |
 | [Stack](https://en.wikipedia.org/wiki/Stack_(abstract_data_type)) | [:white_check_mark:](stack/stack.c) | | [:white_check_mark:](stack/Stack.java) | [:white_check_mark:](stack/stack.py) | [:white_check_mark:](stack/stack.go) | [:white_check_mark:](stack/stack.js) | [:white_check_mark:](stack/Stack.cs) |
+| [Queue](https://en.wikipedia.org/wiki/Queue_(abstract_data_type)) | | | | [:white_check_mark:](queue/queue.py) | | | |
```
Comment (singhpratyush) [inline README.md]: This should be in lexical order.

File: queue/queue.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+class Queue(object):
+
+    def __init__(self):
+        self._list = []
+
+    def count(self):
+        return len(self._list)
+
+    def is_empty(self):
+        if self.count() == 0:
```
Comment (singhpratyush) [inline queue/queue.py]: How about `return self.count() == 0`?

File: queue/queue.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            return False
+
+    def enqueue(self, item):
+        self._list.append(item)
+
+    def dequeue(self):
+        try:
+            return self._list.pop(0)
+        except IndexError:
+            raise IndexError('pop from empty stack')
```
Comment (singhpratyush) [inline queue/queue.py]: This is cool!

File: queue/queue.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+    while queue.count() < 5:
+        print('pushing elements: {0}'.format(n))
+        queue.enqueue(n)
+        n = n + 100
+
+    print('Number of items: {0}'.format(queue.count()))
+    print('Empty queue: {0}'.format(queue.is_empty()))
+
+    while queue.count() >= -20:
```
Comment (singhpratyush) [inline queue/queue.py]: It seems to be a little confusing to put size as -20. You can use `while not queue.is_empty()`, or if you intend to show the exception part, you can use `while True`.

File: queue/queue.py (hunk 2 above)
Comment (amineHorseman) [reply queue/queue.py]: yes, better

File: README.md (hunk 1 above)
Comment (amineHorseman) [reply README.md]: ok

File: queue/queue.py (hunk 4 above)
Comment (amineHorseman) [reply queue/queue.py]: yes, in fact I wanted to show the exception part.
Thank you, I'll use ```while True``` instead

**Pull request merged**